src/heliaServer.ts

```diff
@@ -111,6 +121,10 @@ export class HeliaServer {
       return this.fetchWithoutSubdomain({ request, reply })
     }
     const { ns: namespace, address, '*': relativePath } = request.params as EntryParams
+    if (address.includes('wikipedia')) {
+      await reply.code(500).send('Wikipedia is not yet supported. Follow https://github.com/ipfs/helia-http-gateway/issues/35 for more information.')
+      return
+    }
```

Comment:
disabling wikipedia entries so tiros doesn't spin out on requests since it's not working (theres something special we need to do with HAMT-Sharded directories, and idk what that is right now)